frontend/wallet/src/render/components/Footer.re

```diff
@@ -13,89 +13,82 @@ module Styles = {
-          |> Option.withDefault(~default=pubKey |> PublicKey.toString)
-        )}
+           Tc.Option.andThen(
+             ~f=s => SettingsRenderer.lookup(s, pubKey),
+             settings,
+           )
+           |> Option.withDefault(~default=pubKey |> PublicKey.toString),
+         )}
       </span>
     </div>;
   };
 };
 
-module ActivityLogButton = {
+module PublicKeyButton = {
   [@react.component]
-  let make = () => {
-    <button>
-      <span> {ReasonReact.string("Activity Log")} </span>
-      <span className=Css.(style([marginLeft(`rem(1.0))]))>
-        {ReasonReact.string({j|⌘L|j})}
-      </span>
-    </button>;
-  };
-};
-
-module RightButtons = {
-  module PublicKeyButton = {
-    [@react.component]
-    let make = (~pubKeySelected) => {
-      let str = ReasonReact.string("Copy public key");
-      // The switch is over the <button> rather than within the onClick because
-      // if there exists an onClick the button is no longer disabled (despite
-      // disabled being true), and there is no way to give JSX an `option` for
-      // the click handler.
-      switch (pubKeySelected) {
-      | Some(pubKey) =>
-        <button
-          onClick={_e => {
-            let task =
-              Bindings.Navigator.Clipboard.writeTextTask(
-                PublicKey.toString(pubKey),
-              );
-            Task.perform(task, ~f=()
-              // TODO: Should we toast when this happens? Do we need to handle errors?
-              => Js.log("Copied to clipboard"));
-          }}
-          disabled=false>
-          str
-        </button>
-      | None => <button disabled=true> str </button>
-      };
-    };
-  };
-
-  module SendButton = {
-    [@react.component]
-    let make = () => {
-      <button onClick={_e => Router.navigate(Route.Send)}>
-        {ReasonReact.string("Send")}
-      </button>;
+  let make = (~pubKeySelected) => {
+    let str = ReasonReact.string("Copy public key");
+    // The switch is over the <button> rather than within the onClick because
+    // if there exists an onClick the button is no longer disabled (despite
+    // disabled being true), and there is no way to give JSX an `option` for
+    // the click handler.
+    switch (pubKeySelected) {
+    | Some(pubKey) =>
+      <button
+        onClick={_e => {
+          let task =
+            Bindings.Navigator.Clipboard.writeTextTask(
+              PublicKey.toString(pubKey),
+            );
+          Task.perform(task, ~f=()
+            // TODO: Should we toast when this happens? Do we need to handle errors?
+            => Js.log("Copied to clipboard"));
+        }}
+        disabled=false>
+        str
+      </button>
+    | None => <button disabled=true> str </button>
     };
   };
+};
 
+module SendButton = {
   [@react.component]
-  let make = (~pubKeySelected) => {
-    <div> <PublicKeyButton pubKeySelected /> <SendButton /> </div>;
+  let make = () => {
+    <button onClick={_e => Router.navigate(Route.Send)}>
```

Comment:
I guess we'll convert this to being a modal rather than a route later?

Reply:
Yeah I'd assume so.